.github/workflows/refresh_counts.yml

```diff
@@ -41,5 +41,4 @@ jobs:
           pip install -r requirements.txt
 
       - name: Refresh counts
-        run: |
-          python backend/db/refresh_dataset_group_tables.py --dataset-group counts
+        run: make refresh-counts
```

Comment:
## Split up tables
@Sigfried @Leskena2 I split up the "refresh vocabulary and counts tables" into 2 actions "refresh vocab" and "refresh counts" as Lisa wisely suggested.

## Still having trouble getting either to run
### Counts
Failing for a strange new reason. [Unable to read parquet](https://github.com/jhu-bids/TermHub/actions/runs/6476787183/job/17586030329).
FYI I also tried to see if I could read a parquet file as bytes directly from the `response`. It seems like pandas supports that, but I couldn't get it to work.

### Vocab
Failing for same reason. Getting some message which is not telling me why it is failing. Assume it has something to do with memory or disk space, but from what I can see so far, neither limitation is met. I also heard there may be some kind of CPU limitation, but I have no way to check what that limitation might be, assuming what I've heard is correct. I can continue troubleshooting, but may be in our best interest to pay for premium GitHub runners. I could be wrong, but preliminary estimate is $12/mon.

## Keeping `refresh_voc_and_counts.yml` for now
@Sigfried We have a lot of actions, and you wanted to cull the number of them, but I'm loathe to do that, as (1) it is also useful to have many actions, and (2) it allows us to go back and visit logs of these old actions.
I realize that '2' is not a good reason to keep long term, so I'd like to keep the combined `refresh_voc_and_counts.yml` action temporarily at least until these 2 new actions (vocab and counts) are running in stable fashion.